scrape/scrape.go

```diff
@@ -1663,6 +1695,43 @@ const (
 	scrapeBodySizeBytesMetricName = "scrape_body_size_bytes" + "\xff"
 )
 
+var (
```

Comment:
Whether we want metadata for report samples should be a separate PR imo, and start with no metadata for now